### DISCUSSION TITLE
How many students are there in the science department? 🥼🔬

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1279] [Galaxy Trivia] [answered]
Within the school, the science department has three disciplines.

In total, 280 students study chemistry, 254 students study physics, and 280 students study biology.

In total, 97 students study chemistry and physics, 138 students study physics and biology, and 152 students study chemistry and biology.

73 students study all three disciplines.

**Comment by tomerf** [comment]
There are total of 573 students.

**Comment by peterbaccaro** [comment, marked as answer]
There are **500** students in the science department.

- We know that 73 students study all three disciplines.
- 97 students study chemistry and physics, so 97 - 73 = 24 study just chemistry and physics.
- 138 students study physics and biology, so 138 - 73 = 65 study just physics and biology.
- 152 students study chemistry and biology, so 152 - 73 = 79 study just chemistry and biology.
- 280 students study chemistry, so 280 - 73 - 24 - 79 = 104 study just chemistry.
- 254 students studied physics, so 254 - 73 - 24 - 65 = 92 study just physics.
- 280 students study biology, 280 - 73 - 65 - 79 = 63 study just biology.
- Total = 104 + 92 + 63 + 24 + 65 + 79 + 73 = **500**

**Comment by vitor-tadashi** [comment]
Using the principle of inclusion-exclusion:
280 + 254 + 280 − 97 − 138 − 152 + 73
